.github/workflows/test_presubmit.yaml

```diff
@@ -15,12 +15,20 @@ jobs:
       - uses: actions/setup-python@v2
         name: Install Python
         with:
-          python-version: '3.8'
+          python-version: '3.9'
 
-      - name: Run test script
+      - name: Install deps
         run: |
-          # TODO: Add flag to disable removal of pins.
-          python $GITHUB_WORKSPACE/build_tools/install_and_test.py
+          python3 -m pip install Filecheck
+
+      - name: Run installation
+        run: |
+          python $GITHUB_WORKSPACE/build_tools/configure.py --use-pinned
+          python3 -m pip install . -f https://github.com/google/iree/releases --force-reinstall .[xla,cpu,test]
+      
+      - name: Run tests
+        run: |
+          build_tools/unittest.sh
```

Comment:
These "update github actions" changes and the unittest script seem unrelated to the GPT2 changes? Could that be split into at least two commits, if not two fully separate PRs? Specifically I'm looking at `build_tools/unittest.sh` and wondering why `models/gpt2/test_jax.py` (or other python tests) is not included there (why have a script that just calls lit?)

Reply:
I needed to change how configuration worked selecting versions and adding GPT-2. Potentially I could remove all the GPT-2 work and commit that in a separate follow up but the original inspiration to these changes was adding GPT-2.